src/sectionproperties/analysis/fea.py

```diff
@@ -55,7 +103,7 @@ def _assemble_torsion(
     return k_el, f_el, c_el
 
 
-@njit(cache=True, nogil=True)  # type: ignore
```

Comment:
This impairs performance as compiled code is not cached. Why not directly define a njit decorator when njit cannot be imported. This solution appears to be overcomplex to me.

Reply:
Now that you mention this, that's an obvious solution thanks! I didn't realise this would impair the caching. If you have time to implement a fix I'd be happy to merge, otherwise I will get to this next week. Cheers!

Reply:
I mean, directly define the decorator on line 31:

```py
def njit(**options):
    def decorator(func):
        def wrapper(*args, **kwargs):
            return func(*args, **kwargs)

        return wrapper

    return decorator
```

Reply:
The rest does not need to be changed, so cache will still work if numba is installed. Ppl download and run once, the functions will be compiled, cached, and reused. Without cache, those functions will be compiled every time when the analysis is performed. Since most ppl do not change the source code, caching could benefit most users I presume.